src/components/card/types.ts

```diff
@@ -2,20 +2,22 @@ import { PillCategory } from '../../types/pill-category';
 import { Tag } from '../../interfaces/tag';
 import { Author } from '../../interfaces/author';
 import { Image } from '../../interfaces/image';
+import React from 'react';
 
 export type CardVariant = 'small' | 'medium' | 'large' | 'list' | 'related';
 
 export interface CardProps {
     authors?: Author[];
-    contentDate: string;
-    updateDate?: string;
+    displayDate: string;
     className?: string;
     description?: string;
     title: string;
+    contentType: PillCategory;
     pillCategory: PillCategory;
```

Comment:
this logic got a bit convoluted, but we do need both of these, `pillCategory` signifies the actual text being displayed in the pill and `contentType` is being used to determine which icon to show. this came up because (I think) events is the first case where the designs called for showing a different text inside the pill than the contentType, so for example the contentType is `Event` but we want to display `User Group Meetup` in the pill

Reply:
Ahhh ok makes sense, thanks!